CGWork/CGWorkView.cpp

```diff
@@ -409,32 +409,75 @@ void CCGWorkView::OnFileLoad()
 
 }
 
+static double calcRotateValue(int val)
+{
+	val = val * parentObject.rotation_sensitivity;
+	val = val % 360; // ??
```

Comment:
doesnt really make a difference since sin(theta) is the same as sin(theta + k*2*pi)